test/visual/stroke_projection.js

```diff
@@ -300,7 +300,7 @@ QUnit.module('stroke projection', (hooks) => {
             },
             golden: `stroke-projection/${
               isPolygon ? 'strokeLineJoin' : 'strokeLineCap'
-            }/${testCase.name}/${strokeLineCap}${
+            }/${testCase.name}-${strokeLineCap}${
```

Comment:
I patched this since tests were failing.
I think it might be best to open a folder for each test.
#8311 fixes the option for creating directories on the fly or should at least